docs/quick-start/cmake.rst

```diff
@@ -4,16 +4,7 @@
 Notes about version of CMake
 ----------------------------
 
-* `3.2.0`_ **Minimum required**
-
-  * New ``continue`` command
-  * New synchronization command ``file(LOCK ...)``
-
-* `3.4.1`_
-
-  * **Buggy**, see `issue #405 <https://github.com/ruslo/hunter/issues/405>`__
-
-* `3.5.0`_ **Minimum for iOS projects**
+* `3.5.0`_ **Minimum required**
```

Comment:
last change from me: add a reference to this PR for interested persons to find out more info

```suggestion
* `3.5.0`_ **Minimum required**

  * Since: `PR #689 <https://github.com/cpp-pm/hunter/pull/689>`__
```